tools/acrn-crashlog/README.rst

```diff
@@ -0,0 +1,124 @@
+ACRN-Crashlog
+#############
+
+Introduction
+************
+
+The ``ACRN-Crashlog`` is a debug tool to collect any useful logs/information after each crash/event occurred on the platform, including hypervisor, Service OS(SOS) and Android as a Guest(AaaG). It provides abundant logs for developers to debug/analyze the crashes. And it provides a flexible way to configure which crash/event to be concerned with xml file.
+
+Building
+********
+
+Build dependencies
+==================
+
+The ``ACRN-Crashlog`` tool depends on the following libraries (build and runtime):
+
+- libevent
+
+- OpenSSL
+
+- libxml2
+
+- systemd
+
+- telemetrics-client-dev (optional, detected at build time)
+
+For more detailed instructions on how to set-up your build environment, please refer to the :ref:`getting_started`.
+
+Build
+=====
+
+To build the ``ACRN-Crashlog``
+
+.. code-block:: console
+
+   make
+
+To Clean the build:
+
+.. code-block:: console
+
+   make clean
+
+Installing
+**********
+
+Install dependencies
+====================
+
+Since ``ACRN-Crashlog`` runs as a probe of `telemetrics-client`_, it needs `telemetrics-client`_ (>=1.6.0) to be available on the system.
+
+Install
+=======
+
+To install the build
+
+.. code-block:: console
+
+   sudo make install
+
+Usage
+*****
+
+``ACRN-Crashlog`` works as a probe of `telemetrics-client`_. It is autostarted, running as daemon and sending the crashlog path to `telemetrics-client`_.
```

Comment:
No need to make all references to telemetrics-client a link.  I'd recommend changing this to:

    ACRN-crashlog works as a probe of the Clear Linux `telemetrics-client`_: 
    it runs as a daemon autostarted when the system boots, and sends
    the crashlog path to the telemetrics-client that records events of interest
    and reports them to the backend using ``telemd`` the telemetrics daemon.

Reply:
Also, the probe is called ``acrnprobe`` not ``ACRN-Crashlog``, isn't it?

Reply:
You're right. I'll correct it in the doc. Thanks for your comments.